modin/pandas/concat.py

```diff
@@ -132,3 +139,25 @@ def concat(
         else:
             result_df.columns = new_idx
     return result_df
+
+
+def _determine_name(objs: list, axis):
+    axis_getter = (lambda df: df.columns) if axis else (lambda df: df.index)
+    name_getter = (
```

Comment:
I prefer this to be `def` instead of setting equal to `lambda`.

Reply:
fixed it